core/frontend/certa.json

```diff
@@ -1,7 +1,7 @@
 {
   // Comments are allowed here!
-  "testBundle": "./lib/test/webpack/bundled-tests.js",
-  "instrumentedTestBundle": "./lib/test/webpack/bundled-tests.instrumented.js",
+  "testBundle": "./lib/cjs/test/webpack/bundled-tests.js",
+  // "instrumentedTestBundle": "./lib/cjs/test/webpack/bundled-tests.instrumented.js",
```

Comment:
left over or not working?

Reply:
not working, spoke with @wgoehrig about this, we'll need to fix this later on. I know he was looking into changing the reporter tools we currently use

Reply:
okay, sounds good. There are a few others we have to fix as well.